RFS/src/main/java/com/rfs/cms/CmsEntry.java

```diff
@@ -3,18 +3,23 @@
 import com.rfs.common.RfsException;
 
 public class CmsEntry {
+    public abstract static class Base {
+        protected Base() {}
+    }
+
```

Comment:
There is no functionality in this class, is something missing?  If not lets remove